driver/fixtures/fixtures_test.go

```diff
@@ -28,7 +28,7 @@ var Suite = &fixtures.Suite{
 			"CPPASTImplicitName",
 			"CPPASTName",
 			"CPPASTCompoundStatement",
-			"CPPASTQualifiedNames",
+			//"CPPASTQualifiedNames",
```

Comment:
Is this meant to be deleted?

Reply:
Is currently disabled, see discussion above with Dennys above what to do with these (the schema expects a list of `Identifiers` but in this driver, and most really, the parts can be other nodes like function calls).

Reply:
Fixed!